ios/Podfile

```diff
@@ -13,7 +14,19 @@ def common_pods
 
   # RN development pods
   config = use_native_modules!
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # Hermes is now enabled by default. Disable by setting this flag to false.
```

Comment:
I think this comment is not valid in our case.

Reply:
I'm gonna remove